modules/rds/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "env_code" {
+
+}
```

Comment:
```suggestion
variable "env_code" {}
```
if there are no arguments inside curly brackets, they should be on a single line. please check the same for all variable files